.travis.yml

```diff
@@ -107,27 +105,6 @@ script:
     # Test packages pararell (xargs -P)
     - go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {}
 
-    # Insert build information into compiled binary using ldflags
```

Comment:
If we remove building binary and subsequent build of container, there will be nothing to be pushed to docker hub in git push.

Building container also tests docker file correctness on it's own.